utils/generate_grammar_tables.py

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/env python
+"""Generates various info tables from SPIR-V JSON grammar."""
+
+from __future__ import print_function
+
+import functools
+import re
+import simplejson as json
```

Comment:
`simplejson` has better error message. It helped me for developing. But yeah, `json` should be fine since the grammar file are syntactically correct now.